### DISCUSSION TITLE
Организация кода ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#70] [Announcements]
# Организация кода ⚡️ Angular с примерами кода

Для того чтобы использование библиотеки Angular NgRx стало оправданным и максимально эффективным, необходимо правильно организовать и структурировать код, формирующий хранилище (Store) приложения

https://angdev.ru/ngrx/code-structure/

**Comment by SimLevin** [comment]
По NgRX Signals будут новые статьи?

**Reply by bndby** [reply]
Пока в планах нет. Если есть что интересное - предлагайте, переведем и добавим.
